tests/java/JavaPMDBearTest.py

```diff
@@ -32,3 +34,11 @@ class Hello {
 JavaPMDBearTest = verify_local_bear(
     JavaPMDBear, valid_files=(good_file,), invalid_files=(bad_file,),
     tempfile_kwargs={'suffix': '.java'})
+
+test_files = os.path.join(os.path.dirname(__file__), 'test_files')
+custom_rule_set = os.path.join(test_files, 'custom_rule_set.xml')
+
+JavaPMDBearTest2 = verify_local_bear(
+    JavaPMDBear, valid_files=(good_file,), invalid_files=(bad_file,),
```

Comment:
hey, this test is insufficient because it'd pass even if the setting would do nothing :/